src/x86/cpuid.c

```diff
@@ -710,7 +713,7 @@ struct topology* get_topology_info(struct cpuInfo* cpu, struct cache* cach, int
   }
 
   switch(cpu->cpu_vendor) {
-    case CPU_VENDOR_INTEL:
+    case CPU_VENDOR_INTEL: {
```

Comment:
This was added to resolve clang compile error

Reply:
I have fixed this in another way (just pushed into master)